org.uqbar.project.wollok.lib/src/wollok/lang/WDictionary.xtend

```diff
@@ -26,7 +26,11 @@ class WDictionary implements JavaWrapper<Map> {
 	
 	def basicGet(WollokObject key) { wrapped.get(getInternalKey(key)) }
 	
-	def put(WollokObject key, WollokObject value) { wrapped.put(key, value) }
+	def put(WollokObject key, WollokObject value) {
+		if (key == null) throw new IllegalArgumentException("You cannot put a null key in a Dictionary") 
+		if (value == null) throw new IllegalArgumentException("You cannot put a null value in a Dictionary")
```

Comment:
If you run it in a REPL console this is what you get:

>>> const mapa = new Dictionary()
a Dictionary[]
>>> mapa.put(2, null)
wollok.lang.Exception: You cannot put a null value in a Dictionary
	at wollok.lang.Dictionary.put(_key,_value) [/wollok.wlk:505]
	at  [/__synthetic0.wpgm:3]
	at  [/__synthetic0.wpgm:3]
	at  [/wollokREPL.wlk:1]

Is it ok?